### PR TITLE
Fix incorrect log

### PR DESCRIPTION
**Pull request by X9VoiD**

### CODE REVIEW

File: OpenTabletDriver.Desktop/DesktopDeviceConfigurationProvider.cs [hunk 1]
```diff
@@ -23,15 +23,12 @@ private IEnumerable<TabletConfiguration> GetTabletConfigurations()
 
             if (Directory.Exists(AppInfo.Current.ConfigurationDirectory))
             {
+                Log.Write("Detect", $"The configuration overrides directory '{AppInfo.Current.ConfigurationDirectory}' exist.", LogLevel.Debug);
```
Comment (Sublimelime) [inline OpenTabletDriver.Desktop/DesktopDeviceConfigurationProvider.cs:26]: Grammar/rephrase:

`$"Configuration overrides exist in '{AppInfo.Current.ConfigurationDirectory}', overriding built-in configs.`

If no rephrase is necessary, just add an S to "exist".

Comment (InfinityGhost) [reply OpenTabletDriver.Desktop/DesktopDeviceConfigurationProvider.cs:26]: Made edits to @Sublimelime's review.

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
Apply @Sublimelime's changes.

File: OpenTabletDriver.Desktop/DesktopDeviceConfigurationProvider.cs (hunk 1 above)
Comment (Sublimelime) [reply OpenTabletDriver.Desktop/DesktopDeviceConfigurationProvider.cs:26]: Thanks, still don't know how to make github do that...

**Review by Sublimelime — APPROVED** [review]

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**